auto_py_to_exe/web/css/main.css

```diff
@@ -174,6 +174,10 @@ div[id*="section"] .content {
 
 #current-command textarea {
     width: 100%;
+    border-style: solid;
+    border-width: 2px;
+    border-color: #458BC6;
+    border-radius: 4px;
```

Comment:
I feel these styles should match `button, input, select` to be consistent, as well as the styles added for `#output textarea`.

Reply:
I was reviewing that earlier today. Gonna fix.